### DISCUSSION TITLE
twin.macro with esbuild

### DISCUSSION THREAD

**Discussion by bumi001** [ben-rogerson/twin.macro#619] [Ideas]
Both styled-components and tailwindcss seem to work with esbuild. Wondering if we can nudge twin.macro along that direction as well?

https://github.com/evanw/esbuild/issues/1133#issuecomment-822036385
https://github.com/tailwindlabs/tailwindcss/blob/master/package.json

**Comment by ben-rogerson** [comment]
Hey there, thanks for asking about this.
I'm keen on exploring the possibility of an esbuild version this year.

For now, twin can be used alongside esbuild by loading babel within an esbuild plugin.
Recently a member of the community has shared [an example webpack config](https://gist.github.com/michelson/996ce4b3c05867aaa1b80f6f315724a1) using this setup.
